services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -660,6 +664,93 @@ testScimCreateVsUserRef = do
       (call . delete $ brig . paths ["i", "users", toByteString' uid])
         !!! const 202 === statusCode
 
+testCreateUserTimeout :: TestSpar ()
+testCreateUserTimeout = do
+  env <- ask
+
+  email <- randomEmail
+  scimUser <- randomScimUser <&> \u -> u {Scim.User.externalId = Just $ fromEmail email}
+  (_owner, tid) <- call $ createUserWithTeam (env ^. teBrig) (env ^. teGalley)
+  tok <- registerScimToken tid Nothing
+
+  testFlow tid tok scimUser email
+  -- creating the same user again should behave the same
+  testFlow tid tok scimUser email
```

Comment:
this time, though, the invitation should be accepted, and the user should successfully be created, no?

Reply:
Ah I think I've understood your verbal test spec wrong here:

> "sending the same post /scim/Users request a second time will have the same effect as the first time"

I'll update the test

Reply:
Ok, sorry, you did follow the spec correctly.  But I still like what I *meant* more than what I actually wrote.  :)